include/nng/supplemental/nanolib/conf.h

```diff
@@ -310,6 +310,14 @@ struct conf_exchange {
 	conf_exchange_node **nodes;
 };
 
+#if defined(SUPP_PLUGIN)
+typedef struct conf_plugin conf_plugin;
+struct conf_plugin {
+	char **path;
+	size_t path_sz;
+};
+#endif
```

Comment:
Structure `conf_plugin` should align with the structure of it in config.

for example. the structure of plugin in configuration is 
```
plugin {
    libs = [{
        path = "/path/to/plugin_alpha.so"
    }, {
        path = "/path/to/xxx.so"
    }]
}
```

Then the structure of conf_plugin should be 
```
struct conf_plugin_lib {
    char *path;
}
struct conf_plugin {
    struct conf_plugin_lib **libs;
    size_t len;
}
```

Reply:
Got it, done